Makefile

```diff
@@ -1,19 +1,24 @@
 .PHONY: default
-default: all
+default: help
+
+.PHONY: help
+help:
+	@echo "make help              Show this help message"
+	@echo "make dev               Run the app in the development server"
+	@echo "make lint              Run the code linter(s) and print any warnings"
+	@echo "make test              Run the unit tests"
+	@echo "make docs              Build docs website and serve it locally"
+	@echo "make checkdocs         Crash if building the docs website fails"
+	@echo "make clean             Delete development artefacts (cached files, "
+	@echo "                       dependencies, etc)"
+
+.PHONY: dev
+dev: build/manifest.json
+	gulp watch
 
-.PHONY: all
-all: build/manifest.json
-
-## Remove build artifacts
-.PHONY: clean
-clean:
-	rm -f node_modules/.uptodate
-	rm -rf build
-
-## Run test suite
 .PHONY: test
 test: node_modules/.uptodate
-	yarn test
+	yarn run test
```

Comment:
[yarn test](https://yarnpkg.com/en/docs/cli/test) is an actual command, though just a shortcut for `yarn run test`. Was there any particular reason for the change?

Reply:
I _thought_ this made it run the test just once and then finish, rather than waiting, but that seems not to be the case. For consistency with `make test` in other projects it _should_ run the tests and then exit, which I think would also mean that `make test` could be used to run the tests on Travis and Jenkins. It's nice to have a single `make test` command that's used everywhere because if you ever need to change it, you just change what `make test` does and that gets used everywhere.

Reply:
For now I've just changed this back to `yarn test`